ShipManagementService.Infrastructure/Migrations/20180522192119_InitialCreate.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace ShipManagementService.Infrastructure.Migrations
+{
+    public partial class InitialCreate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Customers",
+                columns: table => new
+                {
```

Comment:
You are missing an customer Identifier. The email is outdated and is replaced by CustomerId.

Reply:
fixed